data_diff/diff_tables.py

```diff
@@ -239,12 +236,31 @@ def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_coun
         for res in self._thread_map(list, diff_iters):
             yield from res
 
+    def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_count=None):
+        logger.info(
+            ". " * level
+            + f"Diffing segment {segment_index}/{segment_count}, key-range: {table1.start_key:x}..{table2.end_key:x}"
+        )
+
+        (count1, checksum1), (count2, checksum2) = self._threaded_call("count_and_checksum", [table1, table2])
+
+        if not (count1 or count2):
+            logger.warn(
+                "Uneven distribution of keys detected. "
```

Comment:
We're not really detecting an uneven distribution of keys, but rather very large key segments where we're not seeing any rows, due to gaps.

Reply:
The gaps exist due to an uneven distribution..

Reply:
Yeah ok sorry lol 

What I mean to say I think it's a bit more user-friendly to talk about it in terms of gaps having few keys...